mmengine/runner/runner.py

```diff
@@ -1683,6 +1683,10 @@ def train(self) -> nn.Module:
             self._train_loop.iter,  # type: ignore
             self._train_loop.max_iters)  # type: ignore
 
+        # Maybe compile the model according to options in self.cfg.compile
+        # This must be called **AFTER** model has been wrapped.
+        self._maybe_compile()
```

Comment:
It could be better to add a `NOTE` in Runner's docstring to tell users how to enable `torch.compile`